scripts/zones/Garlaige_Citadel/Zone.lua

```diff
@@ -35,6 +35,9 @@ zoneObject.onInitialize = function(zone)
     GetMobByID(ID.mob.SERKET):setRespawnTime(math.random(900, 10800))
 
     xi.treasure.initZone(zone)
+
+    SetServerVariable('[Escort]Wanzo', 0) -- Set escort for hire servervariable to 0
```

Comment:
is there anyway we can make this variable a local var dedicated to their relative zones? That might make this a bit cleaner when it comes to how many files this variable needs to touch. 

I think the goal of the quest framework is to put everything into one concise file. Anyway we can pursue that the better imo.